contracts/Cornichon.vy

```diff
@@ -23,21 +23,43 @@ name: public(String[64])
 symbol: public(String[32])
 decimals: public(uint256)
 balanceOf: public(HashMap[address, uint256])
+nonces: public(HashMap[address, uint256])
 allowances: HashMap[address, HashMap[address, uint256]]
+version: public(String[32])
```

Comment:
Make Version a string constant, and have a view-only method that returns the constant to save gas

Reply:
Should I rather remove it altogether and make the domain separator just (chain.id, self)?